engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -440,7 +440,9 @@ class WorkflowActor(workflowToStart: WorkflowToStart,
 
         workflowOptions.get(FinalWorkflowLogDir).toOption match {
           case Some(destinationDir) =>
-            pathBuilders.map(pb => workflowLogCopyRouter ! CopyWorkflowLogsActor.Copy(workflowId, PathFactory.buildPath(destinationDir, pb)))(ec)
+            pathBuilders
```

Comment:
If you're paranoid about other `PathBuilderFactories` behaving similarly in the future, you could possibly try doing `Future(pathBuilders).flatten` here.

OTOH if you manually audit the path builders and this was the only one with a route to throwing exceptions, I think you're good here.